storage.conf

```diff
@@ -30,7 +30,7 @@ size = ""
 #mount_program = "/usr/bin/fuse-overlayfs"
 
 # OverrideKernelCheck tells the driver to ignore kernel checks based on kernel version
```

Comment:
Lets just remove this code chunk from the storage.conf.  Since their is no code that does anything with it.